tests/unit_tests/dft/include/compute_inplace_real_real.hpp

```diff
@@ -94,11 +113,34 @@ int DFT_Test<precision, domain>::test_in_place_real_real_buffer() {
                              oneapi::mkl::dft::config_value::REAL_REAL);
         commit_descriptor(descriptor, sycl_queue);
 
-        sycl::buffer<PrecisionType, 1> inout_re_buf{ input_re.data(), sycl::range<1>(size) };
-        sycl::buffer<PrecisionType, 1> inout_im_buf{ input_im.data(), sycl::range<1>(size) };
-
-        oneapi::mkl::dft::compute_forward<descriptor_t, PrecisionType>(descriptor, inout_re_buf,
-                                                                       inout_im_buf);
+        std::vector<PrecisionType> host_inout_re(size, static_cast<PrecisionType>(0));
+        std::vector<PrecisionType> host_inout_im(size, static_cast<PrecisionType>(0));
+        std::copy(input_re.begin(), input_re.end(), host_inout_re.begin());
+        std::copy(input_im.begin(), input_im.end(), host_inout_im.begin());
+
+        sycl::buffer<PrecisionType, 1> inout_re_buf{ host_inout_re.data(), sycl::range<1>(size) };
+        sycl::buffer<PrecisionType, 1> inout_im_buf{ host_inout_im.data(), sycl::range<1>(size) };
+
+        try {
+            oneapi::mkl::dft::compute_forward<descriptor_t, PrecisionType>(descriptor, inout_re_buf,
+                                                                           inout_im_buf);
+        }
+        catch (oneapi::mkl::unimplemented &e) {
+            std::cout << "Skipping test because: \"" << e.what() << "\"" << std::endl;
+            return test_skipped;
+        }
+
+        {
+            auto acc_inout_re = inout_re_buf.template get_host_access();
+            auto acc_inout_im = inout_im_buf.template get_host_access();
+            std::vector<FwdOutputType> input_data(size, static_cast<FwdOutputType>(0));
```

Comment:
naming

Reply:
addressed this in 1e725f265fb587b92b7786ffc9cb9942bb7b2e7f